packages/rocketchat-slashcommands-msg/client.coffee

```diff
@@ -0,0 +1,14 @@
+RocketChat.slashCommands.add 'msg', (command, params, item) ->
+	trimmedParams = params.trim()
+
+	username = trimmedParams.slice(0, trimmedParams.indexOf(' '))
+
+	if username is ''
+		return
+	username = username.replace('@', '')
+
+	if Session.get('showUserInfo') is username
+		Session.set('showUserInfo', null)
+,
```

Comment:
What's the motivation for clearing `showUserInfo` when messaging the selected user?